app/scripts/modules/core/src/presentation/HoverablePopover.tsx

```diff
@@ -17,21 +23,75 @@ export interface IProps {
 export interface IState {
   popoverIsOpen: boolean;
   target?: any;
+  PopoverRenderer?: React.ComponentClass<PopoverProps> | React.StatelessComponent<PopoverProps>;
 }
 
 @autoBindMethods
 export class HoverablePopover extends React.Component<IProps, IState> {
   public static defaultProps: Partial<IProps> = {
     placement: 'top',
-    id: UUIDGenerator.generateUuid()
+    id: UUIDGenerator.generateUuid(),
```

Comment:
@anotherchrisberry Should we standardize on trailing commas? Most of the codebase in non-trailing so I've just been keeping style, but I like it for diff and copy/paste purposes. And ^ this guy sure seems like to like them.